mutiny-core/src/auth.rs

```diff
@@ -24,7 +24,7 @@ struct CustomClaims {
 
 pub struct MutinyAuthClient {
     pub auth: AuthManager,
-    lnurl_client: Arc<LnUrlClient>,
+    pub lnurl_client: Arc<LnUrlClient>,
```

Comment:
nack

not only should you not be reusing an internal client to `MutinyAuth` for this, but you should be using an internal client to an internal lnurl client library for this either. either have some generic http client that everyone uses at the top level or let things be as is.